cms/static/sass/elements/_system-feedback.scss

```diff
@@ -72,6 +72,60 @@
   }
 }
 
+.page-banner {
```

Comment:
The latter. This is the first time I've tried to show page-level messages in Studio, so I needed to style them.